Adyen/UI/Form/Items/FormImageItem/FormImageItem.swift

```diff
@@ -52,12 +55,10 @@ internal class FormImageView: FormItemView<FormImageItem> {
                                   compatibleWith: nil)
         imageView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            self.heightAnchor.constraint(greaterThanOrEqualToConstant: item.size?.height ?? 46.0),
-            self.widthAnchor.constraint(greaterThanOrEqualToConstant: item.size?.width ?? 46.0),
```

Comment:
Width constraint still make sense - it allows `imageView` to remain bigger than image size.

Reply:
Fixed: 32a3cdc9cc0d9e1ff71bdd9886176033095764dd